transform/snowflake-dbt/models/finance/arr_transactions.sql

```diff
@@ -46,6 +51,26 @@ with mrr as (
     where ls.issued_date is not null
         and ls.edition in ('Cloud Professional')
         and es.reason is null
+)*/
+
+with mrr as (
+   select
+    customer as stripeid,
+    customers.name as customer_name,
+    subscriptions.id as subscription_id,
+    cws_dns as customer_dns,
+    cws_installation as license_id,
+    subscriptions.created::date as date_joined,
+    current_period_end::date as active_license_end,
```

Comment:
we are now using the conventions captured here: https://mattermost.atlassian.net/wiki/spaces/DATAENG/pages/2422603777/DWH+Naming+Conventions for dates and timestamp fields.

Reply:
@catalintomai I don't have access to the atlassian page you referenced.  Can you please grant me access.

Reply:
Shared, please lmk if that did not work.

Reply:
@catalin, I was able to access with my mattermost email.   Adopted the date convention and now closed